newrelic/main.tf

```diff
@@ -1,3 +1,7 @@
+provider "newrelic" {
+  
```

Comment:
Should we be specifying a version here?

Reply:
version is dictated/pinned by whatever the latest version is supported by terraformer's New Relic